.github/workflows/ci.yml

```diff
@@ -94,6 +94,10 @@ jobs:
       matrix:
         arch: [x64]
     steps:
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '5.0.x'
```

Comment:
it's not clear why this CI step is using `setup-dotnet` but none of the other CI steps are using it. Add a comment saying why you're using it

Reply:
Thanks, fixed.